linera-execution/src/evm/database.rs

```diff
@@ -241,3 +242,52 @@ where
         Ok(keccak256(number.to_string().as_bytes()))
     }
 }
+
+impl<Runtime> DatabaseRuntime<Runtime>
+where
+    Runtime: BaseRuntime,
+{
+    pub fn get_block_env(&self) -> Result<BlockEnv, ExecutionError> {
+        let mut runtime = self.runtime.lock().expect("The lock should be possible");
+        // The block height being used
+        let block_height_linera = runtime.block_height()?;
+        let block_height_evm = U256::from(block_height_linera.0);
+        // The coinbase receiving the minted ethereum in Proof Of Work.
+        // Not relevant anymore
+        let coinbase = address!("00000000000000000000000000000000000000bb");
```

Comment:
In[ the docs](https://reth.rs/docs/reth/revm/revm/context/struct.BlockEnv.html) it says
> This is the receiver address of all the gas spent in the block.

which I can imagine this being useful for us at some point in the future.

Let's rename to _beneficiary_ and copy-paste the comment from the REVM as explanation.